core/pseudo.rs

```diff
@@ -13,7 +13,7 @@ pub struct PseudoCursor {
 impl PseudoCursor {
     pub fn new() -> Self {
         Self {
-            current: RefCell::new(None),
+            current: None.into(),
```

Comment:
same here, I don't like using `into` because now I have to jump back to `current` definition to understand what type is going to get converted to

Reply:
I actually do like the `into()` conversions because they tend to make the code less verbose. For me, the IDE shows the type of `current` so that's not really a big problem either. I don't mind the explicit form, though.